extension/src/minn_maxn.rs

```diff
@@ -0,0 +1,1177 @@
+use pgx::{iter::SetOfIterator, *};
```

Comment:
Ideally we would [use the pgx prelude](https://github.com/timescale/timescaledb-toolkit/issues/586) here.

Reply:
This is a little complicated, as some of the code generated by our macros uses pgx types that aren't a part of the prelude (notably `InOutFuncs`, `StringInfo`, and `rust_regtypein`).  We should still make this change, but I think it should be done in a more comprehensive PR that fixes up the macros (hopefully to remove any dependencies on external `use` statements).